src/js/redux/action.js

```diff
@@ -0,0 +1,46 @@
+import Video from '../model/Video.js';
+import {
+  UPDATE_VIDEOS_TO_BE_SHOWN,
+  ADD_SEARCH_TERM,
+  UPDATE_REQUEST_PENDING,
+  INCREASE_SAVED_VIDEO_COUNT,
+} from './actionType.js';
+
+export const updateVideosToBeShown = (items) => {
+  const videos = [];
+
+  items.forEach((item) => {
+    videos.push(new Video(item));
+  });
+
+  return {
+    type: UPDATE_VIDEOS_TO_BE_SHOWN,
+    payload: {
+      videos,
+    },
+  };
+};
```

Comment:
```suggestion
export const updateVideosToBeShown = (items) => {
  return {
    type: UPDATE_VIDEOS_TO_BE_SHOWN,
    payload: {
      videos: items.map(item => new Video(item))
    },
  };
};
```

이정도는 사실 선호의 차이일 수도 있는데, 저는 개인적으로 추가로 변수선언하지않고 작성할 수 있다면 추가 선언을 지양하는 편이예요 👍

Reply:
예전 리뷰어님한테서는 변수로 선언하라는 피드백을 받아서..! 🥶
`map`으로 `videos`에 받아서 넘겨주는 방식으로 코드 길이만 줄였습니다 ㅎㅎ